src/openforms/submissions/api/serializers.py

```diff
@@ -96,6 +96,24 @@ class SubmissionSerializer(serializers.HyperlinkedModelSerializer):
         read_only=True,
     )
 
+    payment_required = serializers.BooleanField(
+        label=_("payment required"),
+        help_text=_("Whether the registration requires payment."),
+        read_only=True,
+    )
+    has_paid = serializers.BooleanField(
+        label=_("user had paid"),
+        source="payment_user_has_paid",
+        help_text=_("Whether the user has completed the required payment."),
+        read_only=True,
+    )
+    payment_amount = serializers.BooleanField(
+        label=_("user had paid"),
+        source="product__price",
+        help_text=_("Amount of payment."),
+        read_only=True,
+    )
```

Comment:
Please group this in a nested serializer holding all the payment details.

Desired JSON response:

```json
{
    "payment": {
         "required": true,
         "completed": false,
         "amount": "123.45"
    }
}
```